jwst/regtest/regtestdata.py

```diff
@@ -253,11 +248,6 @@ def to_asdf(self, path):
         af = asdf.AsdfFile(tree=tree)
         af.write_to(path)
 
-    @classmethod
-    def open(cls, filename):
-        with asdf.open(filename) as af:
-            return cls(**af.tree)
-
```

Comment:
Is there a reason `open()` is removed?  I think this is necessary to parse the `RegtestData` object that gets serialized to ASDF when okifying.

Reply:
was lost in the undoing of the code rearrangement. that operation turned out to be much nastier than it should have been.

Reply:
✅